.gitignore

```diff
@@ -9,6 +9,7 @@
 node_modules/
 npm-debug.log
 yarn-error.log
+yarn.lock
```

Comment:
isn't it recommend to keep it in ? https://classic.yarnpkg.com/lang/en/docs/yarn-lock/ 
ignoring the npm counterpart would make sense though

Reply:
We do no keepthe yarn lock because it would lead to errors when our artifactory urls would be in there. We ignore npm counterparts because currently the lib would not work with npm because of some build commands, we also only have yarn examples in the readme.